x/staking/keeper/msg_server.go

```diff
@@ -688,10 +721,6 @@ func (k msgServer) TokenizeShares(goCtx context.Context, msg *types.MsgTokenizeS
 		return nil, err
 	}
 
-	validator, _ = k.GetLiquidValidator(ctx, valAddr)
```

Comment:
When swapping out the previous LSM logic (here) with new LSM logic (above), why did you move it earlier in the `RedeemTokens` execution path? Same question for the other x/staking keeper function changes

Reply:
The movement wasn't intentional. The intentional part was the bundling of a) checking if the cap is exceeded, with b) modifying progress towards the cap.

But I think moving this is also net positive cause it was a bit hidden down here at the bottom of the function